pymc/distributions/continuous.py

```diff
@@ -1456,12 +1457,11 @@ class Exponential(PositiveContinuous):
 
     Notes
     -----
-    Logp calculation is defined in aeppl.logprob
-    https://github.com/aesara-devs/aeppl/blob/main/aeppl/logprob.py
+    Logp calculation is defined in `aeppl.logprob <https://github.com/aesara-devs/aeppl/blob/main/aeppl/logprob.py/>`_.
 
     Parameters
     ----------
-    lam: float
+    lam : float
```

Comment:
we need to decide on how to document input types to be clear that scalars, arrays and aesara tensors (at least) are valid

Reply:
We have decided to use `tensor_like of float` (or of integer when the parameter should be integer). can you update it?